config/rbac/auth_proxy_service.yaml

```diff
@@ -10,5 +10,9 @@ spec:
   - name: https
     port: 8443
     targetPort: https
+  - name: http
+    protocol: TCP
+    port: 8080
+    targetPort: 8080
```

Comment:
WDYT about using port name reference instead of numeric value? If the pod port changes (yeah, unlikely), the service will not need to.